templates/common/infra/bicep/core/host/aks/acragentpool.bicep

```diff
@@ -0,0 +1,19 @@
+param location string = resourceGroup().location
+param acrName string
+param acrPoolSubnetId string = ''
+
+resource acr 'Microsoft.ContainerRegistry/registries@2021-06-01-preview' existing =  {
+  name: acrName
+}
+
+resource acrPool 'Microsoft.ContainerRegistry/registries/agentPools@2019-06-01-preview' = {
+  name: 'private-pool'
+  location: location
+  parent: acr
+  properties: {
+    count: 1
+    os: 'Linux'
+    tier: 'S1'
```

Comment:
Please make all properties params.  thx

Reply:
FYI - 99% of this bicep comes from the https://azure.github.io/AKS-Construction/ project.  

Bicep lives @ https://github.com/Azure/AKS-Construction/tree/main/bicep